nestjs/src/courses/tasks/tasks.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Body, Controller, Logger, Post, UseGuards } from '@nestjs/common';
+import { ApiForbiddenResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { NotificationsService } from 'src/notifications/notifications.service';
+import { CourseGuard, DefaultGuard, RequiredRoles, Role } from '../../auth';
+import { CheckTasksDeadlineDto } from './dto/check-tasks-deadline';
+import { TasksService } from './tasks.service';
+
+@Controller('tasks')
+@ApiTags('courses tasks')
+@UseGuards(DefaultGuard, CourseGuard)
+export class TasksController {
+  private readonly logger = new Logger('tasks');
+
+  constructor(private tasksService: TasksService, private notificationService: NotificationsService) {}
+
+  @Post('/check-deadlines')
```

Comment:
similar to comment in the prev PR, maybe use `notify` word to make it implicit that it will notify